### ISSUE TITLE
WebGL in an iframe causes the render thread to terminate

### ISSUE THREAD

**Issue by illfang** [cefsharp/CefSharp#1226]
When WebGL content is loaded in an iframe the render thread terminates. I have tried various examples and even the load of just a triangle crashes the render thread.
```
[0828/141524:ERROR:render_process_host_impl.cc(1565)] bad message 132637 terminating renderer.
[0828/141524:ERROR:bad_message.cc(15)] Terminating renderer for bad IPC message, reason 5
```
CefTerminationStatus: ProcessCrashed

The problem can be reproduced with CefSharp.WinForms.Example and CefSharp.Wpf.Example (x86 & x64, Debug & Release) on the master branch at 710db395315c2e66f1093b2180f5b7cd614a98b1 by including an iframe in the home.html

```
diff --git a/CefSharp.Example/Resources/Home.html b/CefSharp.Example/Resources/Home.html
index 8c8e241..8b290d9 100644
--- a/CefSharp.Example/Resources/Home.html
+++ b/CefSharp.Example/Resources/Home.html
@@ -15,6 +15,7 @@
     <link href="assets/css/shCoreDefault.css" rel="stylesheet">
 </head>
 <body>
+    <iframe width="600" height="400" src="http://webglsamples.org/aquarium/aquarium.html"></iframe>
     <a class="sr-only" href="#content">Skip navigation</a>
     <header class="navbar navbar-inverse navbar-fixed-top bs-docs-nav" role="banner">
         <div class="container">
```
The same happend on cefsharp/CefSharp/tree/cef/2272 @ bceb3db4c5e78f956baaf7813349ffa441471afa

When directly accessing the sample without an iframe it is displayed without any problems.

The problem does not occur when the modified Home.html is loaded using the CEF test app (same revision as used by the master): cef_binary_3.2357.1287.g861c26e_windows32_client

OS Name:	Microsoft Windows 8.1 Pro
Version:	6.3.9600 Build 9600
Processor:	Intel(R) Core(TM) i7-4650U CPU @ 1.70GHz, 2301 Mhz, 2 Core(s), 4 Logical Processor(s)
Installed Physical Memory (RAM):	8.00 GB

**Comment by amaitland** [comment]
Does the problem reproduce in the current release version? 

https://github.com/cefsharp/CefSharp/tree/cefsharp/41

**Comment by illfang** [comment]
Yes same behaviour for the 41 release version for CefSharp.WinForms.Example and CefSharp.Wpf.Example (x86, Debug & Release)

cefsharp/41: CefSharp.WinForms.Example x86 debug.log
http://pastebin.com/pnw1JRGU

**Comment by amaitland** [comment]
Can you try with https://github.com/cefsharp/CefSharp.MinimalExample

If it works in `cefclient`, then it's likely something to do with the `JSB` implementation. So testing without that would be helpful.

**Comment by illfang** [comment]
Confirmed correct behaviour for CefSharp.MinimalExample.WinForms and CefSharp.MinimalExample.Wpf (x86 & x64, Debug & Release)

**Comment by illfang** [comment]
As noted in #1263 this issue seems to have been fixed by the changes for #1263

FramedWebGLTest at ~b9bad5ebf3c54cd4f02927373d305ca12d6ed77f
<img width="619" alt="framedwebgltest_at_b9bad5ebf3c54cd4f02927373d305ca12d6ed77f" src="https://cloud.githubusercontent.com/assets/2493562/9900811/42305326-5c9e-11e5-95fd-804523235271.png">

FramedWebGLTest at 5958763c9cb43742419062f6934a8cdd27fb5c47
<img width="618" alt="framedwebgltest_at_5958763c9cb43742419062f6934a8cdd27fb5c47" src="https://cloud.githubusercontent.com/assets/2493562/9900813/45598f5e-5c9e-11e5-9212-abd539bdd950.png">

The issue can be closed.

**Comment by amaitland** [comment]
> The issue can be closed.

Excellent, thanks for the heads up :+1:
